.github/workflows/ci.yml

```
@@ -1,39 +1,29 @@
-name: Scoop Core CI Tests
+name: CI
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)